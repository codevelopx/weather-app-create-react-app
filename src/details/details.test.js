import React from 'react';
import Details from './details';
import { shallow } from 'enzyme';




describe('Details component snapshot', () => {

    it("renders without crashing", () => {
        const details = shallow(<Details />);
        expect(details).toMatchSnapshot();
    });

});


