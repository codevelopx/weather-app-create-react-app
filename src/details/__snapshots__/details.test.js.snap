// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `ShallowWrapper {}`;

exports[`Details component snapshot renders without crashing 1`] = `ShallowWrapper {}`;
